db_scripts/dump.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 
-# For AMI: sudo yum -y install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6-x86_64/pgdg-ami201503-96-9.6-2.noarch.rpm
-#          sudo yum -y install postgresql96
+# export ENV_NAME=prod
+# aws cloudformation create-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts --template-url https://s3.amazonaws.com/crowd-deployment/infrastructure/bastion-hosts.yaml --parameters ParameterKey=EnvironmentName,ParameterValue=$ENV_NAME ParameterKey=KeyPairName,ParameterValue=rstorey@loc.gov --disable-rollback
+# aws cloudformation delete-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts
 
-pg_dump -Fc --clean --create --no-owner --no-acl -U concordia -h $POSTGRESQL_HOST concordia -f concordia.dmp
\ No newline at end of file
+export TODAY=20190129
+export POSTGRESQL_PW=${POSTGRESQL_PW:-password}
+export POSTGRESQL_HOST=${POSTGRESQL_HOST:-localhost}
+export DUMP_FILE=concordia.dmp
+
+echo "$POSTGRESQL_HOST:5432:*:concordia:$POSTGRESQL_PW" >> ~/.pgpass
+chmod 600 ~/.pgpass
+
+pg_dump -Fc --clean --create --no-owner --no-acl -U concordia -h "$POSTGRESQL_HOST" concordia -f "$DUMP_FILE"
+aws s3 cp "$DUMP_FILE" s3://crowd-deployment/database-dumps/concordia.$TODAY.dmp
```

Comment:
Does it make sense to have some sort of guard here which either doesn't run if the environment isn't production or which puts the environment component into the S3 path?